docker/developer_setup_ansible/Dockerfile

```diff
@@ -1,12 +1,12 @@
 
-FROM alpine:3.10
+FROM demisto/crypto:1.0.0.16776
 
 
 # Installing sudo to emulate normal OS behavior...
 RUN apk --update add sudo
 
 # Adding python run time..
-RUN apk --update add python3 openssl ca-certificates
+RUN apk --update add python3 openssl ca-certificates rust
```

Comment:
I don't think you need rust, but I am be wrong. The docker image grew significantly in size from this PR and I think this is related.